docs/_docs/bestpractices/compute/compute.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+parent: Best practices
+permalink: /best-practices/compute
+nav_order: 2
+title: Compute
+author: arclares
+ms.date: 08/16/2024
+ms.service: finops
+description: 'Discover essential FinOps best practices to optimize cost efficiency and governance for your Azure resources.'
+
+---
+
+<span class="fs-9 d-block mb-4">Compute</span>
+Discover essential FinOps best practices to optimize cost efficiency and governance for your Azure compute resources.
+{: .fs-6 .fw-300 }
+
+[Share feedback](#️-looking-for-more){: .btn .fs-5 .mb-4 .mb-md-0 .mr-4 }
+
+<details open markdown="1">
+   <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [🖥️ Virtual Machines](#virtual-machines)
+- [🖱️ Azure Kubernetes Service](#azure-kubernetes-service)
+- [🙋‍♀️ Looking for more?](#️-looking-for-more)
+- [🧰 Related tools](#-related-tools)
+
+</details>
+
+---
+
+## Azure Kubernetes Service
+
+### Query: AKS Cluster
+
+This Azure Resource Graph (ARG) query retrieves detailed information about Azure Kubernetes Service (AKS) clusters within your Azure environment. 
+
+#### Category
+
+Resource management
+
+#### Query
+
+<details>
+  <summary>Click to view the code</summary>
+  <div class="code-block">
+    <pre><code>resources
+  | where type == "microsoft.containerservice/managedclusters"
+  | extend AgentPoolProfiles = properties.agentPoolProfiles
+  | mvexpand AgentPoolProfiles
+  | project
+      id,
+      ProfileName = tostring(AgentPoolProfiles.name),
+      Sku = tostring(sku.name),
+      Tier = tostring(sku.tier),
+      mode = AgentPoolProfiles.mode,
+      AutoScaleEnabled = AgentPoolProfiles.enableAutoScaling,
+      SpotVM = AgentPoolProfiles.scaleSetPriority,
+      VMSize = tostring(AgentPoolProfiles.vmSize),
+      nodeCount = tostring(AgentPoolProfiles.['count']),
+      minCount = tostring(AgentPoolProfiles.minCount),
+      maxCount = tostring(AgentPoolProfiles.maxCount),
+      location,
+      resourceGroup,
+      subscriptionId,
+      AKSname = name

```

Comment:
Can you make sure all the code samples are not indented? The extra spaces will be in the output. If you switch to markdown, this won't be an issue.